app/javascript/bundles/UI/components/Alert/Alert.jsx

```diff
@@ -3,20 +3,18 @@ import React from 'react';
 import css from './Alert.scss';
 import classnames from 'classnames';
 
-var types = ['standard', 'sky', 'fire', 'sun', 'sea'];
+var types = ['standard', 'error', 'warning', 'succes'];
 
 
-const Alert = ({type, content}) => {
+const Alert = ({type, children}) => {
   const classes = classnames('alert', {
     'alert-standard': type == 'standard',
```

Comment:
check out again the list of names because I updated it some minutes ago with additional names.

Palette reference:
https://drive.google.com/file/d/1kENLWBl8E5dbkTLR16GYmrPpNy2sr8oM/view